<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#titleLabel {
background: white;
color: blue;
font-size: 20px;
border: none;
border-bottom:  1px solid black;
padding: 5px;
}

#mainFrame {
border: none;
background: white;
}

#loginFrame {
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ddf, stop: 1 #aaf);
border: 1px solid gray;
padding: 10px;
border-radius: 25px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>聊天室</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="mainFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="styleSheet">
          <string notr="true">background-image: url(:/backgorund.png);</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="chatPage">
          <property name="styleSheet">
           <string notr="true">background-image: url(:/chatpage.png);</string>
          </property>
          <widget class="QTextEdit" name="roomTextEdit">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>40</y>
             <width>551</width>
             <height>351</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QListWidget" name="userListWidget">
           <property name="geometry">
            <rect>
             <x>565</x>
             <y>40</y>
             <width>211</width>
             <height>501</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
          </widget>
          <widget class="QPushButton" name="sayButton">
           <property name="geometry">
            <rect>
             <x>450</x>
             <y>500</y>
             <width>71</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>570</x>
             <y>0</y>
             <width>181</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 63 16pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>已在线用户列表：</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>10</y>
             <width>101</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 63 20pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>聊天室</string>
           </property>
          </widget>
          <widget class="QPushButton" name="exitButton">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>500</y>
             <width>81</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>退出</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>4</x>
             <y>439</y>
             <width>81</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>输入消息：</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="sayLineEdit">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>440</y>
             <width>471</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton">
           <property name="geometry">
            <rect>
             <x>410</x>
             <y>390</y>
             <width>141</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>清除对话框</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="loginPage">
          <property name="styleSheet">
           <string notr="true">background-image: url(:/bg2.png);</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="5" column="1">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>267</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>100</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>232</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="2">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>223</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1" rowspan="2">
            <widget class="QFrame" name="loginFrame">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <property name="spacing">
               <number>20</number>
              </property>
              <item row="2" column="0">
               <widget class="QLabel" name="label_6">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 255);
font: 63 14pt &quot;Ubuntu&quot;;</string>
                </property>
                <property name="text">
                 <string>服务器:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 255);
font: 63 14pt &quot;Ubuntu&quot;;</string>
                </property>
                <property name="text">
                 <string>用户名：</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="usernameLineEdit"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 255);
font: 63 14pt &quot;Ubuntu&quot;;</string>
                </property>
                <property name="text">
                 <string>密码：</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="userpasswordLineEdit">
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="serverLineEdit"/>
              </item>
              <item row="3" column="1">
               <widget class="QPushButton" name="loginButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 28, 8);
font: 63 14pt &quot;Ubuntu&quot;;</string>
                </property>
                <property name="text">
                 <string>登录</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>usernameLineEdit</tabstop>
  <tabstop>userpasswordLineEdit</tabstop>
  <tabstop>roomTextEdit</tabstop>
  <tabstop>userListWidget</tabstop>
  <tabstop>sayLineEdit</tabstop>
  <tabstop>sayButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>sayLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>sayButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>564</y>
    </hint>
    <hint type="destinationlabel">
     <x>768</x>
     <y>570</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>usernameLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>userpasswordLineEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>userpasswordLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>loginButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
